packages/2019-template/src/state/plateau-in-ridership/api.js

```diff
@@ -0,0 +1,20 @@
+import Reduxful from "reduxful";
+import requestAdapter from "../request-adapter";
+
+const apiConfig = { requestAdapter };
+
+const HOST = "https://service.civicpdx.org/transportation-systems";
+
+const apiDesc = {
+  getRidershipOverTime: {
+    url: `${HOST}/passenger-census/system/annual/averages/?format=json`
+  }
+};
+
+const plateauInRidershipAPI = new Reduxful(
+  "plateauInRidershipAPI",
+  apiDesc,
+  apiConfig
+);
+
+export default plateauInRidershipAPI;
```

Comment:
Oh my gosh this pattern is so much cleaner.

An actual question:
I was going to suggest renaming the folder containing this api to template-api-card -- so all of the state for a card lives in a corresponding folder with the same name as the component.

However, if you have an API that you want to use across multiple cards, then keeping a name that corresponded to what the API did could make sense.

How much should live in one Reduxful file?

Reply:
Right now its clean enough that I actually like having it all in one file... I'm down with the renaming. It's much clearer then what corresponds to what.